matterpy/contrib/rss.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+import asyncio
+import feedparser
+
+_conf = None
+_mgr = None
+
+
+def init(manager, conf):
+    global _conf
+    global _mgr
+
+    _conf = conf
+    _mgr = manager
+
+    loop = asyncio.get_event_loop()
+    rss_feeds = [rss_url for rss_url in _conf['rss_feed'].split(',')]
+    for rss_url in rss_feeds:
+        loop.create_task(post_rss(rss_url))
+
+
+async def post_rss(rss_url):
+    n = 0
+    last_message = ''
+    d = feedparser.parse(rss_url)
+    while True:
+        try:
+            for i in d.entries:
+                d = feedparser.parse(rss_url)
+                message = post_to_text(d, n)
+                if last_message < d.entries[n].published:
+                    await _mgr.send(_conf['channel'], message)
+                    last_message = d.entries[n].published
+                    n += 1
+                    if n >= 10:
+                        d = feedparser.parse(rss_url)
+                        n = 0
+            await asyncio.sleep(int(_conf['interval']))
+        except IndexError:
+            d = feedparser.parse(rss_url)
+            continue
+
+
+def post_to_text(feed, n):
+    try:
+        title = feed.entries[n].title
+        link = feed.entries[n].link
+        body = feed.entries[n].description
+    except IndexError:
+        return None
+
+    return "{}, {}, {}".format(title, link, body)
```

Comment:
I suggest putting the format string into the config as well. And use a default that is not just comma-separated. Instead, you can use markdown formatting, and turn the title into a link directly.